docs/ftldns/configfile.md

```diff
@@ -3,63 +3,78 @@ You can create a file `/etc/pihole/pihole-FTL.conf` that will be read by *FTL*DN
 Possible settings (**the option shown first is the default**):
 ### DNS settings
 
-- `BLOCKINGMODE=NULL|IP-NODATA-AAAA|IP|NXDOMAIN`<br>
+- <div class="anchor" id="blocking_mode"></div>
```

Comment:
Does this still work with a self-closing tag?

```suggestion
- <div class="anchor" id="blocking_mode"/>
```

Reply:
No, it does not like that
![image](https://user-images.githubusercontent.com/4417660/51293957-c1ab6e00-19c5-11e9-8dab-ca725b6d1a3d.png)